test/demo/collection/demo_postman_collection.json

```diff
@@ -0,0 +1 @@
ionName\": null,\r\n    \"OrganisationTypeId\": \"DEN\",\r\n    \"OrganisationType\": \"Dentist\",\r\n    \"OrganisationStatus\": \"Visible\",\r\n    \"ServiceType\": \"DEN\",\r\n    \"ServiceSubType\": \"COMPH\",\r\n    \"SummaryText\": \"\",\r\n    \"URL\": \"https:\/\/my-pharmacy.com\/\",\r\n    \"Address1\": \"501-503 MANSFIELD ROAD, NOTTINGHAM\",\r\n    \"Address2\": null,\r\n    \"Address3\": null,\r\n    \"City\": null,\r\n    \"County\": null,\r\n    \"Latitude\": 53.38030624389648,\r\n    \"Longitude\": -1.4826949834823608,\r\n    \"Postcode\": \"NG5 2JJ\",\r\n    \"Phone\": \"0115 9606272\",\r\n    \"Email\": \"health.my-pharmacy@nhs.net\",\r\n    \"Website\": null,\r\n    \"OrganisationSubType\": null,\r\n    \"OrganisationAliases\": [],\r\n    \"OpeningTimes\": []\r\n}","options":{"raw":{"language":"json"}}},"url":{"raw":"https:\/\/uec-dos-integration-di-192.k8s-nonprod.texasplatform.uk\/api\/v1\/nhsuk-event-receiver\/test","protocol":"https","host":["uec-dos-integration-di-192","k8s-nonprod","texasplatform","uk"],"path":["api","v1","nhsuk-event-receiver","test"]}},"response":[]},{"name":"DEMO_EVENT_5 - Time Changes","request":{"method":"POST","header":[{"key":"x-api-key","value":"","type":"text"}],"body":{"mode":"raw","raw":"{\r\n    \"SearchKey\": \"ANEI1245\",\r\n    \"ODSCode\": \"FAT91\",\r\n    \"OrganisationName\": null,\r\n    \"OrganisationTypeId\": \"PHA\",\r\n    \"OrganisationType\": \"Pharmacy\",\r\n    \"OrganisationStatus\": \"Visible\",\r\n    \"ServiceType\": \"PHA\",\r\n    \"ServiceSubType\": \"COMPH\",\r\n    \"SummaryText\": \"\",\r\n    \"URL\": \"https:\/\/my-pharmacy.com\/\",\r\n    \"Address1\": \"501-503 MANSFIELD ROAD, NOTTINGHAM\",\r\n    \"Address2\": null,\r\n    \"Address3\": null,\r\n    \"City\": null,\r\n    \"County\": null,\r\n    \"Latitude\": 53.38030624389648,\r\n    \"Longitude\": -1.4826949834823608,\r\n    \"Postcode\": \"NG5 2JJ\",\r\n    \"Phone\": \"0115 9606272\",\r\n    \"Email\": \"health.my-pharmacy@nhs.net\",\r\n    \"Website\": null,\r\n    \"OrganisationSubType\": null,\r\n    \"OrganisationAliases\": [],\r\n    \"OpeningTimes\": [{\r\n      \"Weekday\": \"Monday\",\r\n      \"Times\": \"09:15-12:45\",\r\n      \"OffsetOpeningTime\": 540,\r\n      \"OffsetClosingTime\": 780,\r\n      \"OpeningTimeType\": \"General\",\r\n      \"AdditionalOpeningDate\": \"\",\r\n      \"IsOpen\": true\r\n    },\r\n    {\r\n      \"Weekday\": \"Monday\",\r\n      \"Times\": \"14:00-17:30\",\r\n      \"OffsetOpeningTime\": 840,\r\n      \"OffsetClosingTime\": 1050,\r\n      \"OpeningTimeType\": \"General\",\r\n      \"AdditionalOpeningDate\": \"\",\r\n      \"IsOpen\": true\r\n    }]\r\n}","options":{"raw":{"language":"json"}}},"url":{"raw":"https:\/\/uec-dos-integration-di-192.k8s-nonprod.texasplatform.uk\/api\/v1\/nhsuk-event-receiver\/test","protocol":"https","host":["uec-dos-integration-di-192","k8s-nonprod","texasplatform","uk"],"path":["api","v1","nhsuk-event-receiver","test"]}},"response":[]}]}]}
```

Comment:
Can this be formatted so its readable?

Also if you havent already please can you add the x-correlation-id header to each of the api requests, should be different for each of the scenarios

Reply:
- Now updated to include correlation id key 'x_correlation_id' in the header on all scenarios.
- Readable format for the collection json fails git commit as githooks seems to have problems with the spacing/tabs and would only pass when collection json is alligned in a list format.

Reply:
Are there any other formatting scripts/online tools you could use that pass the checks. This isn't readable for review in a meaningful way.

Reply:
Json format issue has now been resolved in latest commit.